src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -1,19 +1,45 @@
 (ns puppetlabs.puppetdb.http.command
   (:require [puppetlabs.puppetdb.command.constants :refer [command-names]]
             [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [clojure.string :as str]
             [clojure.tools.logging :as log]
             [puppetlabs.puppetdb.command :as command]
             [puppetlabs.puppetdb.http :as http]
+            [puppetlabs.puppetdb.cheshire :as json]
             [net.cgrand.moustache :as moustache]
             [puppetlabs.puppetdb.middleware :as mid]
             [compojure.core :as compojure]))
 
+(def min-supported-commands
+  {"replace catalog" 6
+   "replace facts" 4
+   "store report" 5
+   "deactivate node" 3})
```

Comment:
My problem with the `min-supported-commands` is just that I could see users wanting something like a `/meta/v1/support_commands` endpoints, or even something in the `/pdb/meta/v1` endpoints like we've talked about, that would give you the the commands we support and the versions we support of those commands. This kind of overlaps with that idea but can't really share code with it.